code/modules/events/masshallucination.dm

```diff
@@ -3,4 +3,17 @@
 
 /datum/event/mass_hallucination/start()
 	for(var/mob/living/carbon/C in living_mob_list)
-		C.hallucination += rand(100, 250)
\ No newline at end of file
+		C.hallucination += rand(100, 250)
+
+/datum/event/mass_drunk
+	startWhen = 10
+	announceWhen = 0
+
+/datum/event/mass_drunk/start()
+	for(var/mob/living/carbon/C in living_mob_list)
+		C.dizziness = 18
```

Comment:
Make sure there aren't procs to set these properly, simply editing the values entirely can lead to problems